test/potluck_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require './lib/potluck'
+
+class PotluckTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_a_date
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_is_an_empty_array
```

Comment:
when we start of tests with `it`, `it` conventionally refers to the object that you're testing-- in this case, an instance of potluck.  So when you say `it is an empty array`, it sounds like you're saying that a potluck object should be an empty array, which is inaccurate. So a better name for this test would be something like `test_it_has_dishes`, which is more general but completely adequate, or `test_it_starts_with_no_dishes` if you want to be more descriptive.